### PR TITLE
Use correct cct branch for cloud6

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4178,17 +4178,19 @@ function onadmin_run_cct()
...
-                        cct_tests+="+test:func:${test}client"
-                    fi
-                done
+            develcloud6|GM6|GM6+up)
+                checkout_branch=cloud6
                 ;;
         esac
 
+        if iscloudver 6plus; then
+          for test in "nova" "manila"; do
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: travis complains about wrong indent - needs to be 2 more spaces

**Comment by dirkmueller** [comment]
Weird, I did the same change a few days ago but I apparently forgot to push it for a review.

Thanks, +1 once the indentation issue is fixed.

**Comment by aspiers** [comment]
Please could you rebase whilst fixing indentation?

**Comment by dirkmueller** [comment]
+1

**Comment by vmoravec** [comment]
+1

**Comment by jdsn** [comment]
Both gating jobs passed the code change and did the right thing. The failure were later in cct - unrelated.

**Pull request merged**